### PR TITLE
Add image signing with GPG blog

### PR DESCRIPTION
**Pull request by saschagrunert**
Refers to https://github.com/containers/libpod/pull/5495

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
I was thinking of only what I call a small post link.  Just a quick "Sascha Grunert has written a tutorial explaining how to use GPG keys to secure your container images stored in a container repository. {maybe another line or two with high level explanation}  Read all about it [here](https://github.com/containers/libpod/blob/master/docs/tutorials/image_signing.md)"

That way if we need to change things, we can do so on Podman.

**Comment by saschagrunert** [comment]
> I was thinking of only what I call a small post link. Just a quick "Sascha Grunert has written a tutorial explaining how to use GPG keys to secure your container images stored in a container repository. {maybe another line or two with high level explanation} Read all about it [here](https://github.com/containers/libpod/blob/master/docs/tutorials/image_signing.md)"
> 
> That way if we need to change things, we can do so on Podman.

Makes sense, how about this one now?

File: _posts/2020-03-13-image-signing.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: How to sign and distribute container images using Podman
+layout: default
+categories: [blogs]
+tags: containers, images, signing, podman, cri-o, oci, gpg
+---
+
+[Sascha Grunert][0] has written a tutorial explaining how to use Gnu Privacy Guard
+(GPG) keys to secure your container images stored in a container repository.
+Signing container images is nothing magically and can drastically enhance
```
Comment (TomSweeneyRedHat) [inline _posts/2020-03-13-image-signing.md]: magically -> magicall

**Comment by TomSweeneyRedHat** [comment]
One typo, otherwise LGTM
Thanks @saschagrunert 
And note to self, need to wait for the libpod PR to merge

File: _posts/2020-03-13-image-signing.md (hunk 1 above)
Comment (saschagrunert) [reply _posts/2020-03-13-image-signing.md]: Fixed :+1:

**Comment by saschagrunert** [comment]
@TomSweeneyRedHat I think this can be merged not since the related PR in containers/podman got merged, too.

**Pull request merged**